README.md

```diff
@@ -284,6 +284,31 @@ At this point, the power tool is only able to move an arbitrary number of messag
 JustSaying.Tools.exe move -from "source_queue_name" -to "destination_queue_name" -in "region" -count "1"
 ````
 
+## Sample Application
+
+- To run the sample application against a simulated AWS SQS / SNS endpoint run this container
+    ```sh
+    docker pull pafortin/goaws
+    docker run -d --name goaws -p 4100:4100 pafortin/goaws
+    ```
+- Alternatively to use your real AWS account
+    - Locate the setup code `services.AddJustSaying(...)` in both `Program.cs` and `Startup.cs`
```

Comment:
At a later point, we should link these directly to the relevant code using permalinks.

Reply:
Done. I've just linked to the files, not the lines in question as I find that a bit to subject to breaking as code churns